docs/registry.md

```diff
@@ -7,7 +7,8 @@ It contains record type it manages, e.g.:
 
 Prefix and suffix are extended with %{record_type} template where the user can control how prefixed/suffixed records should look like.
 
-In order to maintain compatibility, both records will be maintained for some time, in order to have downgrade possibility.
+In order to maintain compatibility, both records will be maintained for some time, in order to have downgrade possibility.  
```

Comment:
So you mean some intermediate `v0.12.x` version would have to be a mandatory step in the migration plan from before `v0.12.0` to >= `v0.13.0`?

Reply:
I was more thinking of preparing all PRs for v0.13.0, including decommissioning the old TXT format, so that we can avoid having a master branch with things that are mid-migration.

Reply:
I'm fine with `v0.13.0` having only the new format without the migration logic. But that would still necessitate some migration guide for the users coming from `< v0.12.0`, no?